src/status_im2/contexts/quo_preview/list_items/token_value.cljs

```diff
@@ -0,0 +1,57 @@
+(ns status-im2.contexts.quo-preview.list-items.token-value
+  (:require
+    [quo2.core :as quo]
+    [react-native.core :as rn]
+    [reagent.core :as reagent]
+    [status-im2.contexts.quo-preview.preview :as preview]))
+
+(def descriptor
+  [{:label   "Token:"
+    :key     :token
+    :type    :select
+    :options [{:key   :eth
+               :value "ETH"}
+              {:key   :snt
+               :value "SNT"}]}
+   {:label   "State:"
+    :key     :state
+    :type    :select
+    :options [{:key   :default
+               :value "Default"}
+              {:key   :pressed
+               :value "Pressed"}
+              {:key   :active
+               :value "Active"}]}
+   {:label   "Status:"
+    :key     :status
+    :type    :select
+    :options [{:key   :empty
+               :value "Empty"}
+              {:key   :positive
+               :value "Positive"}
+              {:key   :negative
+               :value "Negative"}]}
+   (preview/customization-color-option)
+   {:label "Metrics?:"
+    :key   :metrics?
+    :type  :boolean}])
+
+(defn preview
+  []
+  (let [state (reagent/atom {:token    :snt
+                             :state    :default
+                             :status   :empty
+                             :color    :blue
```

Comment:
We need to replace the `color` key with the `customization-color` key (and inside the component too) as it's the standard key across the codebase.

Additionally,  the `(preview/customization-color-option)` uses `customization-color` key. Any changes in the color will not be reflected in the component.

Reply:
Fixed it. Thanks @smohamedjavid 👍